src/renderer/pages/project/generate.vue

```
@@ -52,8 +52,8 @@
 const checkboxOptions: { key: OptionKey }[] = [
   { key: "movie" },
   { key: "audio" },
-  { key: "pdfSlide" },
-  { key: "pdfHandout" },
+//  { key: "pdfSlide" },
```

ESLint

[error] Insert `··`

(prettier/prettier)